homeworks/iurii.stepanenko_IuriiSky/homework_5/task4.js

```diff
@@ -0,0 +1,28 @@
+const people = [
+    { id: 1, name: 'Nick', friends: [2, 5, 6] },
+    { id: 2, name: 'John', friends: [1, 3] },
+    { id: 3, name: 'Mike', friends: [2, 5] },
+    { id: 4, name: 'Janny', friends: null },
+    { id: 5, name: 'Edward', friends: [1, 3] },
+    { id: 6, name: 'Jeen', friends: [5, 1] },
+];
+
+const getPeople = function showPeople(userId) {
+    const targetPerson = people.find(person => person.id === userId);
+    if (targetPerson === undefined) {
+        return null;
+    }
+    const friendsPeople = [];
+    if (targetPerson.friends === null) {
+        return friendsPeople;
+    }
+    targetPerson.friends.forEach((element) => {
+        const friendPerson = people.find(person => person.id === element);
+        friendsPeople.push(friendPerson);
```

Comment:
here you could just use `filter()` for example

Reply:
I fix (change), maybe this code looks better?